.github/workflows/pr-backstage-plugin-workflow.yaml

```diff
@@ -46,3 +48,14 @@ jobs:
 
       - name: test
         run: yarn workspace @devoteam-nl/open-dora-backstage-plugin test:coverage
+
+      - name: publish
+        working-directory: backstage-plugin/plugins/open-dora
+        run: |
+          # Replace the @devoteam-nl with @devoteamnl in package.json
+          sed -i 's/"@devoteam-nl\//"@devoteamnl\//g' package.json
+          yarn version --new-version 0.0.$PR_NUMBER --no-git-tag-version
```

Comment:
Consider using [gitversion](https://gitversion.net/) for the version as it follows SemVer and works fine for PRs as well. They also got a GH action: https://github.com/GitTools/actions

Reply:
This is a cool tool, though it's highlighted an issue with the current release setup that the version in the github registry is different from NPM registry. Probably worth keeping them in sync by publishing to both on release. For another PR anyway.